client/src/game/carrier.js

```diff
@@ -63,12 +63,30 @@ class Carrier extends EventEmitter {
     this.drawGarrison()
   }
 
+  drawShape() {
+    this._rotateCarrierTowardsWaypoint(this.graphics_colour);
+    switch(this.player.shape) {
+      case 'circle':
+        this.graphics_colour.drawCircle(0, 0, 4)
+        return
+      case 'square':
+        this.graphics_colour.drawRect(-4, -4, 8, 8)
```

Comment:
This should be in its own function like the diamond and hexagon for consistency.